lib/ain-cpp-imports/src/bridge.rs

```diff
@@ -29,17 +29,17 @@ pub mod ffi {
 
         fn getChainId() -> u64;
         fn isMining() -> bool;
-        fn publishEthTransaction(data: Vec<u8>) -> String;
+        fn publishEvmTransaction(data: Vec<u8>) -> String;
```

Comment:
These are ETH transactions, as they are TXs of the format of Ethereum. EVM has not nothing to do with this, nor defines a TX spec.